readme.md

```diff
@@ -203,9 +203,23 @@ Points-free function definitions look just like normal assignments without `func
 
 TODO
 
-## Guarded Functions
+## Guards
+Guards are boolean expressions that prevent evaluation of code unless the guard evaluates to true.
+It can be simulated thanks to the conditional ternary Operator.
```

Comment:
I might be wrong here though. Reading here http://c2.com/cgi/wiki?GuardClause gives me a different impression. Though the definition there may be more about procedural programming.